iiblib/iibclient.py

```diff
@@ -144,7 +157,7 @@ def next(self):
     def prev(self):
         """Load items for previous page and set it as current"""
 
-        if self.page > 0:
+        if self.page > 1:
```

Comment:
IIB starts paging from 1?

Reply:
According to documentation of flask sqlalchemy pages are 1 indexed
https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/#flask_sqlalchemy.Pagination.page